File: src/main/java/com/scaler/tictactoe/models/Cell.java
package com.scaler.tictactoe.models;

public class Cell {
    private Player player;
    private int row;
    private int col;
    private CellState cellState;

    public Player getPlayer() {
        return player;
    }

    public void setPlayer(Player player) {
        this.player = player;
    }

    public int getRow() {
        return row;
    }

    public void setRow(int row) {
        this.row = row;
    }

    public int getCol() {
        return col;
    }

    public void setCol(int col) {
        this.col = col;
    }

    public CellState getCellState() {
        return cellState;
    }

    public void setCellState(CellState cellState) {
        this.cellState = cellState;
    }
}

File: src/main/python/cell.py
class Cell:
    def __init__(self, x, y, player=None) -> None:
        self._x = x
        self._y = y
        self._player = player
    
    
    @property
    def x(self):
        return self._x
    
    @x.setter
    def x(self, value):
        self._x = value
    
    @property
    def y(self):
        return self._y
    
    @y.setter
    def y(self, value):
        self._y = value
    
    @property
    def player(self):
        return self._player
    
    @player.setter
    def player(self, value):
        self._player = value
    